bio_hansel/qc/checks.py

```diff
@@ -41,16 +41,56 @@ def is_missing_tiles(st: Subtype, df: DataFrame, p: SubtypingParams) -> Tuple[Op
     Returns:
         None, None if less missing tiles than tolerate; otherwise, "FAIL", error message
     """
-    if not st.are_subtypes_consistent:
-        return None, None
+    error_status = None
+    error_messages = None
+
+    if st.are_subtypes_consistent:
+        error_status, error_messages = check_for_missing_tiles(is_fastq=st.is_fastq_input(),
```

Comment:
You could return early on the happy path (no inconsistent subtypes):
```python
if st.are_subtypes_consistent:
    return check_for_missing_tiles(...)
```
Also you wouldn't need to initialize `error_status` and `error_messages`

Reply:
Hey, that's an awesome idea! I'll change it.